src/plugins/layout.ts

```diff
@@ -39,13 +45,36 @@ export default class Layout implements IPlugin {
         subtree: true
       });
     }
+    if (this.insertRule) {
+      let that = this;
+
+      // Some popular open source libraries, like styled-components, optimize performance
+      // by injecting CSS using insertRule API vs. appending text node. A side effect of
+      // using javascript API is that it doesn't trigger DOM mutation and therefore we
+      // need to override the insertRule API and listen for changes manually.
+      CSSStyleSheet.prototype.insertRule = function(style, index) {
```

Comment:
>CSSStyleSheet [](start = 6, length = 13)

Is this always available? Should we not detect this is supported? And also wrap in a try/catch block as this is a browser compat area.